roles/jboss_fuse_on_karaf/tasks/main.yml

```diff
@@ -96,3 +96,66 @@
   become: true
   when: 'jboss_fuse'
 
+- name: Use locate to look for camel-ver
+  raw: locate camel-core | sed -n 's/.*\(redhat-[0-9]\{6\}\).*/\1/p'
+  register: jboss_fuse_camel_ver
+  ignore_errors: yes
+  when: 'have_locate and jboss_fuse'
+
+- name: check /system/org/apache/camel/camel-core for camel-ver
+  raw: ls -1 "{{ item }}"/system/org/apache/camel/camel-core 2>/dev/null | sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
+  register: jboss_fuse_on_karaf_camel_ver
+  ignore_errors: yes
+  with_items: "{{ karaf_homes }}"
+  when: 'jboss_fuse'
+
+- name: set the camel-ver fact
+  set_fact:
+      fuse_camel_version: "{{
+      (jboss_fuse_camel_ver.get('stdout_lines', []) +
+       jboss_fuse_on_karaf_camel_ver.get('stdout_lines', [])) | unique }}"
+  ignore_errors: yes
+  when: 'jboss_fuse'
+
+- name: Use locate to look for activemq-ver
+  raw: locate activemq | sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
+  register: jboss_fuse_activemq_ver
+  ignore_errors: yes
+  when: 'have_locate and jboss_fuse'
+
+- name: check /system/org/apache/activemq/activemq-camel for activemq-ver
+  raw: ls -1 "{{ item }}"/system/org/apache/activemq/activemq-camel 2>/dev/null | sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
```

Comment:
Based on @noahl bash "globbing" identified issue (#1089)
I think this should be:

`ls -1 '{{ item }}/system/org/apache/activemq/activemq-camel' 2>/dev/null`